index.bs

```diff
@@ -595,6 +595,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
         to |authenticator|'s attachment modality, [=iteration/continue=].
     1. If {{AuthenticatorSelectionCriteria/requireResidentKey}} is set to |true| and the |authenticator| 
         is not capable of storing a [=Client-Side-Resident Credential Private Key=], [=iteration/continue=].
+    1. If {{AuthenticatorSelectionCriteria/requireUserVerification}} is set to |true| and the
```

Comment:
@jyasskin is right.  The client doesn't know whether the authenticator can do user verification or not.  This needs to be passed on to the authenticator - not acted upon by the client.